source/borderrouter_thread_tasklet.c

```diff
@@ -147,10 +149,14 @@ static int thread_interface_up(void)
     device_config.PSKd_len = len;
     memset(device_config.PSKd_ptr, 0, len + 1);
     memcpy(device_config.PSKd_ptr, param, len);
+    rf_read_mac_address(rf_mac);
+    memcpy(device_config.eui64, rf_mac, 8);
 
-    thread_link_configuration_get(&link_setup);
+    channel_list.channel_mask[0] = MBED_CONF_APP_RF_CHANNEL_MASK;
```

Comment:
Channel list should be NULL
The correct place for these settings are in the active configuration and stack should but default to all.

This entire parameter is questionable in node init and should be deprecated